src/test/test_subman.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+TMP=/tmp/subman$$
```

Comment:
maybe could use `mktemp --tmpdir -d` ? so we can respect the `$TMPDIR`, which can in turn be overrided by user the make system.